src/Makefile

```diff
@@ -129,6 +130,18 @@ endif
 # Include paths to dependencies
 FINAL_CFLAGS+= -I../deps/hiredis -I../deps/linenoise -I../deps/lua/src
 
+ifneq ($(NO_SYSTEMD),1)
```

Comment:
I'd perhaps use `USE_SYSTEMD` here, following preexisting style? Also `yes`/`no` instead of `0`/`1`.